### PR TITLE
fix(ui): ban (and some other icons) vertically aligned

### LINKED ISSUE #193: Center icon for placements on candidates grid
**Problem**
On Candidates page: icon for Disqualified or Reserved placement is not aligned with text:

CleanShot 15 11 2024 at 12-05

**Solution**
Center vertically icons relative to text.

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #193.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/assets/stylesheets/src/shared/_scaffolds.scss [hunk 1]
```diff
@@ -199,6 +199,7 @@ picture {
 
 .icon {
   stroke-width: 1.25;
+  vertical-align: middle;
```
Comment (ksnitsky) [inline app/assets/stylesheets/src/shared/_scaffolds.scss:202]: It's not exactly vertical in the center, but the default was `bottom` (for some reason) and it's definitely more centered

We can also set a numeric value and -2px for 15px icons is perfect in the center, sort of, but for larger icons a different number is needed and it looks like a HACK

Comment (ksnitsky) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: With middle:
image

-2px:
image

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: maybe `sub` ?

https://github.com/user-attachments/assets/5ff7797f-24cc-4a4b-8510-4e2b997f5d28

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: or even rethink compose logic to used grid system for this column

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: seems `sub` is not the solution either

Comment (ksnitsky) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: Yes, I originally tried via flex, but then with a long position name it's not clear where to position this block

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: i'm afraid Amgalan won't approve this solution

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:202]: using pixel hack is also bad idea.

Comment (ksnitsky) [reply app/assets/stylesheets/src/shared/_scaffolds.scss]: With sub in firefox it looks the same as with hack
image

But in chromium sub and middle are the same

Comment (ksnitsky) [reply app/assets/stylesheets/src/shared/_scaffolds.scss]: Anyway, it's better than it was

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss]: for me `sub` looks not ok for the last item
image

Comment (wrmk) [reply app/assets/stylesheets/src/shared/_scaffolds.scss]: but let's try this solution

**Review by wrmk — APPROVED** [review]

**Pull request merged**